### PR TITLE
Mongo db

### PR DESCRIPTION
**Pull request by FreytagDom**

### CODE REVIEW

File: models/DayTimeFactor.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const factorSchema = new Schema({
+  morningfactor: { type: Number, required: true },
+  lunchfactor: { type: Number, required: true },
+  eveningfactor: { type: Number, required: true },
+});
```
Comment (ThomasKirst) [inline models/DayTimeFactor.js:9]: Based on your question on the discussion board I would amend your Schema like this:
```suggestion
const factorSchema = new Schema({
  _id: { type: Number },
  morningfactor: { type: Number, required: true },
  lunchfactor: { type: Number, required: true },
  eveningfactor: { type: Number, required: true },
});
```

File: pages/api/insulinFactor/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    case 'GET':
+      const daytimeFactors = await getAllFactors();
+      return response.status(200).json({ daytimeFactors });
+
+    case 'POST':
+      await dbConnect();
+
+      const data = request.body;
+
+      await DayTimeFactor.create(data);
```
Comment (ThomasKirst) [inline pages/api/insulinFactor/index.js]:
```suggestion
      const result = await DayTimeFactor.findOneAndUpdate({ _id: 1 }, data, { new: true, upsert: true });
      console.log(result, 'RESULT')
```

**Review by ThomasKirst — CHANGES_REQUESTED** [review]
Hi Dominik,
regarding your [question](https://github.com/orgs/neuefische/discussions/228) I suggested two small changes.
Apart from that your PR looks pretty huge for a review 😅

File: pages/api/insulinFactor/index.js (hunk 2 above)
Comment (FreytagDom) [reply pages/api/insulinFactor/index.js]: Hy Thomas, 
Thanks, I have just found a good solution that even works and was just about to write you an answer.

**Pull request merged**